test/functional/readconcern_tests.js

```diff
@@ -60,44 +60,26 @@ describe('ReadConcern', function() {
     metadata: { requires: { topology: 'replicaset', mongodb: '>= 3.2' } },
 
     test: function(done) {
-      var listener = require('../..').instrument(function(err) {
-        test.equal(null, err);
-      });
 
       // Contains all the apm events
-      var started = [];
+      let started = [];
```

Comment:
All of these `let`s can be `const`s because their values are never changing. We set `let configuration` and we don't ever assign a new value to `configuration` so it should be a `const`.